android/src/main/java/io/parity/signer/components/IdentIcon.kt

```diff
@@ -16,6 +18,12 @@ import io.parity.signer.models.intoImageBitmap
 @Composable
 fun IdentIcon(identicon: List<UByte>, size: Dp = 28.dp) {
 	Image(
-		identicon.intoImageBitmap(), stringResource(R.string.description_identicon), modifier = Modifier.size(size)
+		identicon.intoImageBitmap(),
+		stringResource(R.string.description_identicon),
+		modifier = Modifier
+			.size(size)
+			.clip(
+				RoundedCornerShape(100)
```

Comment:
Circle? it's 
CircleShape (which is RoundedCornerShape(50) )
I would do it in the same line (but no problem, can always realign)